src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -1,5 +1,30 @@
 [
   {
+    "name": "Data Loss Prevention",
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/dlp/compute.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Compute Engine API",
+      "compute":"java",
+      "site":"https://cloud.google.com/compute/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/dlp/v2beta1/apidocs/com/google/cloud/compute/package-summary.html",
+      "infotip":"The com.google.cloud.dlp.v2beta1 packages",
```

Comment:
Editorial aside: I really don't like these package names: it makes updating to a new version painful even when there is no API breakage